doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -122,6 +122,19 @@ The {PROCESS_ENGINE} now supports the singleton timer start node in a process wi
 
 endif::PAM[]
 
+== New wrapper modules
+
+{PRODUCT} {ENTERPRISE_VERSION} now provides new wrapper modules to make the Drools engine lighter and more maintainable. There is an ongoing refactoring process to isolate and descope optional features that are not related to the internal algorithm of the engine. Drools engine leverages MVEL for a lot of internal usages like DRL parsing, constraints evaluation, and templates generation. Usages of MVEL are isolated in a new `drools-mvel` module. Drools engine implements KieSessions serialization with `protobuf` to ensure better backward compatibility. This serialization feature is isolated in the `drools-serialization-protobuf` module.
+
+According to your project requirements, you can add `drools-mvel` and `drools-serialization-protobuf` modules in your project classpath.
+
+{PRODUCT} now provides the following new wrapper modules:
+
+* *drools-engine*: Contains `drools-core`, `drools-compiler`, and `drools-model-compiler` dependencies modules
+* *drools-engine-classic*: Contains `drools-core`, `drools-compiler`, and `drools-mvel` dependencies modules
+
+If you want to use the Drools engine along with the executable model, then it will be sufficient to include the `drools-engine` wrapper module in your project dependencies.  But if you want to use the Drools engine without the executable model, you must import your dependencies to the `drools-engine-classic` wrapper module.
```

Comment:
[style] - Remove "then it will be sufficient", and avoid starting a sentence with "But".

Suggestion

s/If you want to use the Drools engine along with the executable model, then it will be sufficient to include the `drools-engine` wrapper module in your project dependencies.  But if you want to use the Drools engine without the executable model, you must import your dependencies to the `drools-engine-classic` wrapper module.
/
To use the Drools engine with the executable model, you must include the `drools-engine` wrapper module in your project dependencies. To use the Drools engine without the executable model, you must import your dependencies to the `drools-engine-classic` wrapper module.

Reply:
This makes more sense. Thank you for the suggestion.